File: src/main/java/SpotifyAuth.java
import java.net.http.HttpRequest;

public class SpotifyAuth {
    private String client_id;
    private String response_type;
    private String redirect_uri;
    private String mainURI;

    // scope som behövs : user-follow-read

    public SpotifyAuth(String redirect_uri){
        mainURI = "https://api.spotify.com/v1";
        client_id = "ee0bdca0269146eaa8345787f079f9d2";
        response_type = "code";
        this.redirect_uri = redirect_uri;
        connectToSpotify();
    }

    private void connectToSpotify() {
       // get("/authorize")

    }


}

File: src/main/resources/public/script/spotify-auth.js
async function getToken(){
    // Funktionen hämtar spotifys token och sparar denna lokalt i webbläsaren.
    let params = (new URL(location.href.replace('#','?'))).searchParams;
    let token = params.get('access_token');
    window.localStorage.setItem('access_token',token)   
    location.href = "listArtists.html";
}
    
function login(){
    // Funktionen hanterar inloggningen på spotifys konto. 
    let scope = 'user-follow-read';
    let clientID = '7a6d44b7b4c7435f9ed4340b9e613395';
    let redirect_uir = 'http://localhost:8888/callback';
    let response_type = 'token';
    
    window.location.href = `https://accounts.spotify.com/authorize?client_id=${clientID}&redirect_uri=${redirect_uir}&response_type=${response_type}&scope=${scope}`;
}
